### ISSUE TITLE
[BUG] Vehicles/furnitures bugged when changing material 

### ISSUE THREAD

**Issue by LoneDev6** [PluginBugs/Issues-ItemsAdder#769]
**Describe the bug**
Basically if you change material from a block (example: STONE) to an item (example: PAPER), regenerate the pack and then place an already crafted item it will spawn an entity with vanilla texture instead of the new one.

**To Reproduce**
0. craft an furniture or vehicle
1. change material from a block (example: STONE) to an item (example: PAPER) of a furniture or vehicle
2. regenerate the pack
3. place the already crafted old item
4. see the issue

**Expected behavior**
spawning the new model instead of the "bugged" entity

**Info (please complete the following information):**
 - Server version: 1.16.5 paper
 - Client version: 1.16.5
 - Plugin version: 2.1.4
 
**Solution**
A "good" solution would be to check if the items have different material compared to the item in registry each time an inventory is open, and then update them if they don't match.

**Solution possible downsides**
Basically it could generate some server overhead as checking each item each time is not very good as I have to check NBT of the each item.

**Another solution**
Sync the item material and custommodeldata only when the furniture is placed. This fixed just the current bug and not all the other cases, but should work well and won't cause any high CPU usage.
